City-of-Pori/pori-kada
# PORIJATE-15 Styling

To test:

1. drush fra -y && drush cc all
2. Go to waste search and use it with all kinds of browsers and see if it works well on all sizes.

ATTENTION: Does not include waste type page styling.
